awslabs/performance-dashboard-on-aws
# Dashboard Listings Tab Accessibility

## Description

Add accessibility functionality to dashboard listings tabs

- Ability to tab to core tabs (Drafts, Publish Queue, etc.)
- Add keyboard functionality: Ability to select tab using 'Enter'
- Aria Label for screen reader will read: "Dashboard Tab {Name}"
- Aria Label will indicate currently selected tab with "Active Dashboard Tab {Name}"

## Testing
Can test it here: https://d2wo27gatac4jo.cloudfront.net/admin/dashboards

- Turn on screen reader and tab through the core dashboard tabs
- Press Enter to select highlighted tab
- When tabbing to active tab. Can confirm Aria Label indicates this is the active tab

-Added and passed Unit Testing

---

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
